pkg/platform/github.go

```diff
@@ -20,13 +20,25 @@ import (
 	"time"
 
 	"github.com/google/go-github/v53/github"
+	"github.com/sethvargo/go-retry"
 	"golang.org/x/oauth2"
 
 	gh "github.com/abcxyz/guardian/pkg/github"
 	"github.com/abcxyz/pkg/githubauth"
+	"github.com/abcxyz/pkg/logging"
 )
 
-var _ Platform = (*GitHub)(nil)
+var (
+	_ Platform = (*GitHub)(nil)
+
+	ignoredStatusCodes = map[int]struct{}{
```

Comment:
Did this list come from anywhere? If so, could you link the docs/source/etc? I'm imagining a future maintainer wondering whether to update this or if it needs these exact values.

Reply:
I added a doc comment. The status codes are defined in each endpoint https://docs.github.com/en/rest/pulls/review-requests?apiVersion=2022-11-28#request-reviewers-for-a-pull-request--status-codes.

I reduced the list to only those that are used by AssignReviewers, but it should be updated when more implementations are added.